src/ProjectNeon/Assets/Z_Tests/Features/Effects/InterceptAttackTests.cs

```diff
@@ -0,0 +1,31 @@
+﻿
+using NUnit.Framework;
+using UnityEngine;
+
+public sealed class InterceptAttackTests
+{
+    [Test]
+    public void InterceptAttack_ApplyEffect_AttackIsIntercepted()
```

Comment:
This test doesn't check the health of the original target.

Reply:
@SilasReinagel Fixed